Authentication/Extensions.swift

```diff
@@ -2,20 +2,42 @@
 //  Extensions.swift
 //  Authentication
 //
-//  Created by Daniela Riesgo on 3/7/16.
+//  Created by Daniela Riesgo on 7/25/16.
 //  Copyright © 2016 Wolox. All rights reserved.
 //
 
 import Foundation
 
 public extension String {
     
-    var localized: String {
-        return NSLocalizedString(self, tableName: nil, bundle: NSBundle.mainBundle(), value: "", comment: "")
+    public var frameworkBundle: NSBundle { return  NSBundle(forClass: LoginView.self) }
+    
+    /**
+         Returns a localized representation of the string,
+         searching first in the main bundle or if it does
+         not have the key in the framework one if.
+     */
+    public var frameworkLocalized: String {
```

Comment:
Also, I think this should be named localized, rather than `frameworkLocalized`. Frameworklocalized make me think that it searches the localization just from the framework, but as you did (very good) it looks for it in main bundle first and then in the framework, so it should be just localized. What do you think? @danielaRiesgo @guidomb